src/stactools/lila_hkh_glacier/cog.py

```diff
@@ -0,0 +1,64 @@
+import logging
+
+from subprocess import CalledProcessError, check_output
+
+logger = logging.getLogger(__name__)
+
+
+def create_cog(
```

Comment:
Should we use the cogify function on https://github.com/stac-utils/stactools/blob/main/src/stactools/core/utils/convert.py?